internal/resource/sysfs-lib.go

```diff
@@ -21,11 +21,18 @@ import (
 	"k8s.io/klog/v2"
 )
 
-type vfioLib struct{}
+type vfioLib struct {
+	nvpci.Interface
+}
 
 // NewVfioManager returns an resource manger for devices with VFIO PCI driver
 func NewVfioManager() Manager {
-	return &vfioLib{}
+
+	nvpcilib := nvpci.New()
```

Comment:
Do we need to pass the driver root here?

Reply:
What do you mean by driver root?

Reply:
I mean whether one of the `Options`:
```
// Option defines a function for passing options to the New() call.
type Option func(*nvpci)

// WithLogger provides an Option to set the logger for the library.
func WithLogger(logger logger) Option {
	return func(n *nvpci) {
		n.logger = logger
	}
}

// WithPCIDevicesRoot provides an Option to set the root path
// for PCI devices on the system.
func WithPCIDevicesRoot(root string) Option {
	return func(n *nvpci) {
		n.pciDevicesRoot = root
	}
}

// WithPCIDatabasePath provides an Option to set the path
// to the pciids database file.
func WithPCIDatabasePath(path string) Option {
	return func(n *nvpci) {
		n.pcidbPath = path
	}
}
```
that `nvpci.New()` accepts is required. I suppose since we're always running in a container we don't expect a different root for pci devices or a different pcidb path.

Reply:
Yes we dont expect both of the paths to be different, so not setting them.